doc/knowledge-base.md

```diff
@@ -0,0 +1,194 @@
+			'/api/markdowns': {
+				target: "http://127.0.0.1:3000",
+				pathRewrite: { '^/api/markdowns': '/markdowns'}
+			}
+		}
+	}
+```
+
+Note that `target`'s port and `port` may be different in your case, but the port in target should be the same as port which you define in `webpackDevServer`. In our test example we use port `3000` so our `target`'s port is also `3000` - `http://127.0.0.1:3000`.
+
+### JSON file with paths and titles
+
+`KnowledgeBase` component uses entry json file with defined paths and titles for your markdown files. In our example we call that file `index.json`.
+This `index.json` file should contain array (list) of objects (dictionaries) with paths and titles to each markdown file. `KnowledgeBase` component will automatically read this file and render navigation bar based on order of objects inside the array and it's titles. Component also take first item's path in `index.json` list and render content of that file on this path. Every time user will click on another item in navigation bar component will take path of that item, load file on that path and render it's content.
+
+Take a look on our example of how `index.json` file look in our getting started app:
+
+```
+[
+	{
+		"title": "Welcome",
+		"path": "/test/welcome.md"
+	},
+	{
+		"title": "Test 1",
+		"path": "/test/test-1.md"
+	},
+	{
+		"title": "Test 2",
+		"path": "/test/test-2.md"
+	}
+]
+```
+
+On this step our `public` folder looks like this:
+```
+- public
+  -- content
+    --- test
+      ---- welcome.md
+      ---- test-1.md
+      ---- test-2.md
+```
+
+## 2. Creating container and routes
+
+### Creating container
+
+In your application in file which will contain container with `KnowledgeBase` component you need to import `KnowledgeBase` from `asab-webui` and return it in your component with `app` prop from props of your container: 
+
+```
+import React from 'react';
+
+import { KnowledgeBase } from 'asab-webui';
+
+const KnowledgeBaseContainer = (props) => {
+	return (
+		<KnowledgeBase app={props.app} />
+	);
+}
+```
+
+Also we should add their `api` which we configured earlier and `entryPath` for our `index.json` file.
+
+`KnowledgeBase` takes `api` and create axios instance with `app.AxiosCreate()` method we get from asab-webui application. As we configured our api to be `content` then we pass it as string to `api` prop of `KnowledgeBase`.
+
+As we mentioned above `KnowledgeBase` takes `index.json` file to constuct itself and load content of markdown files. To fetch this `index.json` file `KnowledgeBase` needs to know path to it. As we have put our `index.json` file into `/content/test/` and we've already defined our api as `content` then we need to provide `entryPath` as `/test/index.json`. 
+
+Now our container should look like this:
+
+```
+import React from 'react';
+
+import { KnowledgeBase } from 'asab-webui';
+
+const KnowledgeBaseContainer = (props) => {
+	return (
+		<KnowledgeBase
+			app={props.app}
+			api="content"
```

Comment:
@titkov49 `api` might be misleading here. I would recommend to use something like `apiPath` or something with informative value

Reply:
@Pe5h4 `api` changed to `apiPath`